lesson_05/homework_05.py

```diff
@@ -37,3 +72,20 @@
 # а значення - списки імен людей, які потрапляють в кожен діапазон.
 # Приклад виводу:
 # {'10-19': ['A'], '20-29': ['B', 'C', 'D'], '30-39': ['E'], '40-49': ['F']}
+new_dict = dict(person_list)
+print(new_dict)
+age_dict = {"10-19":[], "20-39":[], "30-39":[], "40-49":[]}
+for key, value in person_list:
+    if 10<v<19: 
```

Comment:
Прям очевидна помилка
```suggestion
    if 10<value<19: 
```

Reply:
Та я над нею півночі страждала, засмутилась, що не вийшо і махнула рукою навіть на очевидне не звернула увагу( Можете, будь ласка, підказати яке там мало бути рішення